apps/demo/src/app/events-sample/events-sample.component.ts

```diff
@@ -0,0 +1,196 @@
+import { CommonModule } from '@angular/common';
+import { Component, inject } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { MatButtonModule } from '@angular/material/button';
+import { MatCardModule } from '@angular/material/card';
+import { MatChipsModule } from '@angular/material/chips';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatGridListModule } from '@angular/material/grid-list';
+import { MatIconModule } from '@angular/material/icon';
+import { MatInputModule } from '@angular/material/input';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { injectDispatch } from '@ngrx/signals/events';
+import { bookEvents } from './book-events';
+import { BookStore } from './book.store';
+
+@Component({
+  selector: 'demo-events-sample',
+  standalone: true,
+  imports: [
+    CommonModule,
+    FormsModule,
+    MatCardModule,
+    MatButtonModule,
+    MatInputModule,
+    MatFormFieldModule,
+    MatIconModule,
+    MatChipsModule,
+    MatGridListModule,
+    MatToolbarModule,
+  ],
+  template: `
+    <mat-toolbar color="primary">
+      <span>Book Store with Event Tracking</span>
+    </mat-toolbar>
+
+    <mat-card>
+      <mat-card-content>
+        <mat-form-field appearance="outline">
+          <mat-label>Search books</mat-label>
+          <input
+            matInput
+            [(ngModel)]="filterText"
+            (ngModelChange)="dispatch.filterUpdated({ filter: $event })"
+            placeholder="Filter by title or author..."
+          />
+          <mat-icon matSuffix>search</mat-icon>
+        </mat-form-field>
+
+        <button mat-raised-button color="primary" (click)="addRandomBook()">
```

Comment:
nit: though the demo is lowkey, I would like things like requiring a `type` on `button` elements like this. I am going about backfilling this for existing demo code.